datalad_gooey/resource_provider.py

```diff
@@ -32,14 +32,19 @@ def get_icon(self, name):
             return QIcon()
         icon = self._icons.get(name)
         if icon is None:
-            icon = QIcon(str(
-                self._ressource_path / 'icons' / f'{name}.svg'))
+            icon = QIcon(str(self.get_icon_path(name)))
             self._icons[name] = icon
         return icon
 
     def get_best_icon(self, label):
         icon_name = GooeyResources.label_to_name.get(label)
         return self.get_icon(icon_name)
 
+    def get_icon_path(self, name):
+        return self.get_resource_path('icons') / f'{name}.svg'
+
+    def get_resource_path(self, category):
+        return self._ressource_path / category
```

Comment:
I see traces of German spelling in the variable name, but no problem since it's internal :wink:

Reply:
LOL. I de-germanified it. Thx!